### PR TITLE
Added copy table button per flowcell lane

### PR DESCRIPTION
**Pull request by sofiahag**
.....as suggested by Pär.

However, this only copies the "sample table" values, not the "total" value headers above the main tables, because that is a separate table, and I'm not sure how to copy two table id's in one click?

I don't know how much of a priority it is to be able to copy the total values, I'll have to check with Pär, in that case maybe an additional button would have to be created?

### CODE REVIEW

**Comment by sofiahag** [comment]
Up on stage!

**Review by parlundin — APPROVED** [review]
Tack Sofia!

File: run_dir/static/js/flowcell.js [hunk 1]
```diff
@@ -4,6 +4,19 @@ function display_undetermined(lane){
...
 
+// Copy flowcell lane table to clipboard
+$('.lane-copy').click(function(){
+    var lane_btn = '#' + $(this).attr('id');
+    var clipboard = new Clipboard(lane_btn);
+    clipboard.on('success', function(e) {
+        e.clearSelection();
+        $(lane_btn).addClass('active').html('<span class="glyphicon glyphicon-copy"></span> Copied!');
+        setTimeout(function(){
+        $(lane_btn).removeClass('active').html('<span class="glyphicon glyphicon-copy"></span> Copy table');
```
Comment (alneberg) [inline run_dir/static/js/flowcell.js:15]: Super tiny thing, but the icon is lost when copied once, it should be `fa` not `glyphicon`. This bug is probably present in a lot of places
```suggestion
        $(lane_btn).removeClass('active').html('<span class="fa fa-copy"></span> Copy table');
```

Comment (sofiahag) [reply run_dir/static/js/flowcell.js:15]: Ah, you're right!

**Pull request merged**